.github/CONTRIBUTING.md

```diff
@@ -58,11 +58,15 @@ This library will load a file from the `config` folder, based on an environment
 ### Setting up the config
 
 1. Create a file named `development.json` at [../config/](../config/).
-2. Copy the contents of [../config/staging.json](../config/staging.json).
-3. In your `.env` file set `NODE_CONFIG_ENV` to `development`.
-4. Change the `guildId` field to your server's ID.
+2. Paste the contents of [../config/staging.json](../config/staging.json) into `development.json`.
+4. In your `development.json` file replace the `guildId` field with your server ID.
 5. Update the channel and role IDs to those in your server.
-6. Add your own user ID and that of your test bot to `modLogsExclude`.
+6. Add your own user ID and that of your test bot to `modLogExclude `.
+
+### Setting up the env
```

Comment:
Setting up the env should be a separate item. Not under 'Using different configurations'. The only overlap between the env and configs would be the `NODE_CONGIF_ENV`.

If wanting to have a separate entry on how to set up the env, I would suggest moving it to before the 'Using different configurations'. So there we tell the user to copy the env example, and update tokens if they wish to use certain features.

Then in the relevant sections for the config and setting up the bot we can keep the "In your `.env` file set `NODE_CONFIG_ENV` to `development`." and "Inside the .env file, type `BOT_SECRET=TOKEN` replacing TOKEN with what you copied in `step 6.`" respectively.